gLasso/gLasso.py

```diff
@@ -42,14 +43,14 @@ def gLasso_model(features, labels, mode, params):
         return tf.estimator.EstimatorSpec(mode, predictions=predictions)
 
     # Compute loss.
-    loss = tf.losses.mean_squared_error(labels=tf.case(tf.expand_dims(labels, axis=1), tf.float32),
```

Comment:
I think maybe you were trying to use `tf.cast`?

Anyways, you probably don't need to explicit type conversion and reshaping here. 

It's better to convert the shape of `response` instead. and cast `label` to float32 when generating the data (see my version of simu_data.py line 67)